build.py

```diff
@@ -54,6 +54,10 @@ def run(cmd):
         f"-DCMAKE_INSTALL_PREFIX={install_dir}/{project} {defines} {shared_cmake} {charmander_header_cmake}")
     run(f"cmake --build {build_dir}/{project} --config Release")
     run(f"cmake --install {build_dir}/{project} --config Release")
+if platform == "windows":
+    run(f"{build_dir}/squirtle/Release/hello.exe")
```

Comment:
```
dumpbin /DEPENDENTS hello.exe
```
should tell us which dlls are directly linked to the executable (and we would *not* be expecting transitive dll dependencies that are not used directly, same as on Linux/mac)

Reply:
Sure, I can add it, I was mainly focusing on the Linux case, but I will add it too

Reply:
Added.